content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/check_refreshed.rb

```diff
@@ -20,33 +20,45 @@ def main
 
               private
 
-              def task
+              def update_task(message)
+                return unless service_action == 'Provision'
```

Comment:
@tinaafitz 
Can we not check service_action but rather check if we have a task we update the message else skip over